content/support/changelog/peer-to-peer/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: layout-sidebar
+name: Peer-to-Peer (Beta)
+description: Information on additions and changes for the Peer-to-Peer API (Beta).
+order: 100
+published: true
+showInNav: true
+icon: fa fa-share-alt
+back_to_top: true
+title: Peer-to-Peer (Beta) Changelog
+---
+
+# {{ name }}
+
+Monitor this page to keep up with the [Peer-to-Peer API (Beta)]({{ stache.config.peer_to_peer_entity_reference }}) latest changes and {{ stache.config.api_type_name }} service releases.
+
+## 2017-03-28
+
+### New
+
+The Peer-to-Peer API has been released for a public beta. This API is used to create supporter pages, and query campaigns, donors, donations, and teams.
```

Comment:
Minor wording suggestion: In the second sentence, I'd delete "is used to" and change "create" to "creates", delete the comma after "pages", and change "query" to "queries."